model_read_only/migrations/14.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,31 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+            SELECT ir_model_id, res_groups_id
+            FROM ir_model_group_readonly_rel_tmp;
+        """,
+    )
+    data = env.cr.fetchall()
+    grouped = {}
+    for item in data:
+        key = item[0]
+        value = item[1]
+        if key in grouped:
+            grouped[key].append(value)
+        else:
+            grouped[key] = [value]
+    for k, v in grouped.items():
```

Comment:
Please don't use short name for variables, this one not readable. Better, if you use understandable names for variables. For example:
```py
for key, values_list  in grouped.items():
    ...
```